python/servo/platform/linux.py

```diff
@@ -127,6 +127,16 @@ def _platform_bootstrap(self, force: bool) -> bool:
         installed_something |= self._platform_bootstrap_gstreamer(force)
         return installed_something
 
+    def linker_flag(self) -> str:
+        # the rust-lld binary downloaded by rustup
+        # doesn't respect NIX_LDFLAGS and also needs
+        # other patches to work correctly. Use system
+        # version of lld for now
```

Comment:
consider mentioning NixOS/nixpkgs#220717 and filing a NixOS or rustup issue about programs linked with rustup’s lld?

Reply:
Good idea! I've added link to the issue in 4588f8a . I'll create a issue in NixOS for the LDFLAGS stuff.